src/client/tablet_client.cc

```diff
@@ -779,28 +779,6 @@ bool TabletClient::Count(uint32_t tid, uint32_t pid, const std::string& pk, cons
     return Count(tid, pid, pk, idx_name, "", filter_expired_data, value, msg);
 }
 
-bool TabletClient::Count(uint32_t tid, uint32_t pid, const std::string& pk, const std::string& idx_name,
```

Comment:
another `Count` function above will call this function. you can move this implement above.

Reply:
Ignore that previous. I was looking at this pre-coffee. To clarify, it looks like the count function on line 777 should instead use the implementation from the removed Count function, correct?

Reply:
yes,

Reply:
Okay. I've just pushed a new commit, remove the call to Count from Count, and adding back in the implementation. Let me know if anything else is needed!

Reply:
There is compiling errors https://github.com/4paradigm/OpenMLDB/runs/6959539945?check_suite_focus=true#step:7:587
It also need to remove `ts_name` from here https://github.com/4paradigm/OpenMLDB/blob/40d3cbc98eb4a5750078f4849a8f097f6c89a1a1/src/cmd/openmldb.cc#L3587. thank you